src/frontend-proxy.ts

```diff
@@ -0,0 +1,101 @@
+import { getSubdomain, getPathname } from './url-utils'
+
+export async function handleRequest(
+  request: Request
+): Promise<Response | null> {
+  const probability = Number(global.FRONTEND_PROBABILITY)
+  const allowedTypes = JSON.parse(global.FRONTEND_ALLOWED_TYPES)
+
+  const url = request.url
+  const path = getPathname(url)
+
+  if (getSubdomain(url) !== 'de') return null
+
+  if (path === '/enable-frontend') {
+    const response = new Response('Enable frontend')
+
+    setCookieUseFrontend(response, 0)
+
+    return response
+  }
+
+  if (path === '/disable-frontend') {
+    const response = new Response('Disable frontend')
+
+    setCookieUseFrontend(response, 1)
+
+    return response
+  }
+
+  if (path.startsWith('/_next/') || path.startsWith('/_assets/'))
```

Comment:
Just one small addition: `/api/` is necessary for injections and other client-side request to work, could you add it please? Thanks!